src/test/scala/rps/GameTest.scala

```diff
@@ -0,0 +1,47 @@
+package test
+
+import rps.Game.evaluatePlay
+import rps.model.Move
+import rps.model.Move._
+import rps.model.Result._
+import rps.model.Result
+
+class GameTest extends munit.FunSuite {
+
+  case class Play(userMove: Move, computerMove: Move)
+
+  def check(
+      name: String,
+      inputs: List[Play],
+      expected: Result
+  )(implicit loc: munit.Location): Unit = {
+    test(name) {
+      val obtained =
+        inputs.map(play => evaluatePlay(play.userMove, play.computerMove))
+
+      assertEquals(obtained, List.fill(inputs.length)(expected))
```

Comment:
couldn't we use a foreach or something similar, instead of `List.fill`?

Reply:
good point 🔍
I spent a little time on this point, `forEach` was the first approach written, but in second time I thought about the way mUnit shows assertion failures, and then I see that in case of list comparison the diff message is more useful. 

using a comparison between list mUnit assertion failures shows the difference like the following, and this may be useful to identify which element throws test failure.
``` 
26:    "player win",
values are not the same
=> Obtained
List(
  PlayerWin(),
  Draw(),
  PlayerWin()
)
=> Diff (- obtained, + expected)
   PlayerWin(),
-  Draw(),
+  PlayerWin(),
   PlayerWin()
```

meanwhile, the message provided when using `forEach`  is minor expressive and is just a bit tricky to identify the wrong input in the list.

```
26:    "player win",
values are not the same
=> Obtained
Draw()
=> Diff (- obtained, + expected)
-Draw()
+PlayerWin()
```

anyway, the failure diff is not the main topic for this step (and shouldn't happen), furthermore, there may be better ways of writing this test 😄.
@veej thanks for your review, for now, I'll refactor this point with `forEach` method 👍🏻